vagrant.rb

```diff
@@ -17,6 +17,20 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 #####################################################################################
+# AWS vars
+# something like 'xxxxxxxxxxxxxxxxxxxx'
+$aws_key=''
+# something like'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx'
+$aws_secret=''
+# name of your key (region specific)
+$aws_keyname=''
+# your VPC id, like 'subnet-c412ccbf'
+$aws_vpc=''
+
+# security-group id inside your VPC
+# create them in VPC panel => security_group, not in the classic EC2 panel
+$aws_server_group=''
+$aws_agent_group=''
```

Comment:
Those lines should be in the Vagrantfile so that we don't commit them.
The example should be in the default one written by rtf